Ninano/Ninano/Controllers/EventTab/SearchResult/SearchResultViewController.swift

```diff
@@ -16,7 +16,7 @@ final class SearchResultViewController: UIViewController {
     @IBOutlet private weak var eventCollectionView: UICollectionView!
     private var isNotificationButtonSelected = false
     
-    fileprivate let systemImagesTemp = ["performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2", "performanceImage1", "performanceImage2"]
+    var eventData: [Event] = []
```

Comment:
```suggestion
    private var eventList: [Event] = []
```

Data는 맞기에 타입을 연상가능하도록 명명되면 어떨까요 ?

Reply:
감사합니다 !